pkg/api/api.go

```diff
@@ -54,7 +54,9 @@ type API struct {
 	pubkey     string // PEM encoded public key
 	pubkeyHash string // SHA256 hash of DER-encoded public key
 	signer     signature.Signer
-	verifier   *client.LogVerifier
+	// timestamping cert chain
+	certChain []*x509.Certificate
```

Comment:
any chance we could store both this and the PEM encoded version so we don't waste cycles re-encoding?